### PR TITLE
Math Magicians:Full website

### PR DESCRIPTION
**Pull request by Tumworobere**
In this project, I achieved the following:

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Used React basic styling.
- Used React-Router to make the routes work.

### CODE REVIEW

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Tumworobere, 

Excellent work so far! 👌
You have implemented all the required features correctly. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [ ] I can't seem to run your project because of some missing dependencies like `react-router-dom` and wrong/not found files in the styles directory. Please update your project then ask for another CRR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,45 +1,44 @@
+/* eslint-disable import/no-unresolved */
 /* eslint-disable no-unused-vars */
 import React, { useState } from 'react';
-import '../calculator.css';
+import Screen from './Screen.js';
+import Button from './Button.js';
 import calculate from '../logic/calculate.js';
 
 const Calculator = () => {
```
Comment (TedLivist) [inline src/components/calculator.js:8]: - [ ] As stated in the requirements, while you are encouraged to style the app as you like, your general layout of the pages should still match the wireframe. In your **Calculator** page, your calculator is somewhat centered. Kindly move your calculator to the right side of the page to match the wireframe provided.

  Your Implementation | Provided wireframe
  :---:|:---:
  tumwo-des | vicma-calc-temp

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Tumworobere,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @Tumworobere 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

-  n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by Tumworobere** [comment]
> Hi @Tumworobere
> 
> Wooow👏, Your project is complete! There is nothing else to say other than... it's time to merge it. 👏
> 
> Congratulations! 🎉
> ## Optional suggestion
> 
>     * n/a
> 
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


Thanks @Emmyn5600

**Pull request merged**